e2e/src/e2e/Group-Monitor/profiles-and-filter.cy.ts

```diff
@@ -126,7 +126,6 @@ describe('Group-Monitor User', () => {
     cy.get('[data-cy="column-selection"]')
       .contains('Testheft')
       .click();
-    cy.wait(1000);
```

Comment:
Bist du hier sicher, dass diese Wartezeiten nicht wichtig waren, damit es headless stabiler läuft? Nicht dass wir uns da wieder Probleme reinholen.
Finds grundsätzlich gut es zu beschleunigen, wo es geht, aber nicht zum Preis von Instabilität in der CI.